content/docs/reference-javascript-environment-requirements.md

```diff
@@ -23,8 +23,8 @@ ReactDOM.render(
 );
 ```
 
-React also depends on `requestAnimationFrame` (even in test environments).  
-You can use the [raf](https://www.npmjs.com/package/raf) package to shim `requestAnimationFrame`:
+React `requestAnimationFrame` पर भी निर्भर करती है (परीक्षण वातावरण में भी)।
+आप `requestAnimationFrame` के समर्थन के लिए [raf](https://www.npmjs.com/package/raf) package का उपयोग कर सकते हैं :  
```

Comment:
`package` => `पैकेज`

Reply:
in hindi glossary  its is written no translation is required for package

Reply:
Oh. Yes. Totally forgot about it.
Thanks for cross checking